src/Rules/Drupal/Tests/BrowserTestBaseDefaultThemeRule.php

```diff
@@ -71,6 +71,7 @@ public function processNode(Node $node, Scope $scope): array
 
         $excludedTestTypes = TypeCombinator::union(
             new ObjectType('Drupal\\FunctionalTests\\Update\\UpdatePathTestBase'),
+            new ObjectType('Drupal\\FunctionalTests\\Installer\\InstallerConfigDirectoryTestBase'),
             new ObjectType('Drupal\\FunctionalTests\\Installer\\InstallerExistingConfigTestBase')
         );
```

Comment:
We should only add it to the union if Drupal core is > 10.3.

Reply:
It's a brand new class, will it fail if it is in the union and it does not exist?

Reply:
It will not